.github/workflows/main.yml

```diff
@@ -54,14 +54,16 @@ jobs:
         - build_php_table_backend_utils
     runs-on: ubuntu-latest
     if: |
-        contains(fromJson('["success"]'), needs.build_datatypes.result)
+        always()
+        && (contains(fromJson('["success", "skipped"]'), needs.build_datatypes.result) || contains(fromJson('["success", "skipped"]'), needs.build_php_table_backend_utils.result))
     steps:
       - name: Tests passed
-        run: echo "Tests passed" >> $GITHUB_STATE
+        run: echo "Tests passed"
 
   monorepo_split:
     needs: test_results
     runs-on: ubuntu-latest
+    if : startsWith(github.ref, 'refs/tags/')
```

Comment:
Tady mi není jasné, jaktože je to tady skipped - https://github.com/keboola/storage-backend/actions/runs/4197084002/jobs/7279473951

Protože předtím "test_results" je true. A startsWith taky true. Takže by to mělo proběhnout, ne ?

Reply:
Keď to nieje tag tak netreba púšťať Split nie ? Robi to branch v tej splitnutej libke bez tagu ?

Reply:
jj dělá to i branche - abys mohl u libovolné package udělat keboola/package:dev-CT-123 třeba

Reply:
Ale možná to není potřeba.

Reply:
Príde mi že je to lepšie s tagom ale teda kľudne to zmaz.

Reply:
Skončili jsme na "if it works, it ain't stupid", ale furt nerozumím tomu proč to prochází :(